src/icon-manifest/supportedFolders.ts

```diff
@@ -1,33 +1,31 @@
 /* tslint:disable max-line-length */
-
 import { IExtensionCollection, IFolderExtension, FileFormat } from '../models/IExtension';
 
 export const extensions: IExtensionCollection<IFolderExtension> = {
   supported: [
-    { icon: 'aws', extensions: ['aws'], format: FileFormat.svg },
-    { icon: 'aws', extensions: ['aws'], dot: true, format: FileFormat.svg },
+    { icon: 'aws', extensions: ['aws', '.aws'], format: FileFormat.svg },
```

Comment:
Ah, yes. That's because they can only use simple types in vscode settings. That's why I changed format field to union type.